pyproject.toml

```diff
@@ -16,3 +16,10 @@ pytest = "^8.1.1"
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.ruff]
+line-length = 100
+
+[tool.ruff.format]
+quote-style = "single"
```

Comment:
I think this is fine. Especially with python3.6+ support.

However, black is a double quote enforcer. 🤷